### DISCUSSION TITLE
Cannot figure out how to use different themes in different virtual desktops on Windows 11

### DISCUSSION THREAD

**Discussion by zhichaoh** [t1m0thyj/WinDynamicDesktop#418] [Q&A] [answered]
I downloaded the latest release (4.7.0) as the release note says the virtual desktop is supported on Windows 11. I am using Windows 11 (build 22000.493). I am trying to select different theme after I switched to a different virtual desktop. However, all virtual desktop still use the same theme. 

How can I take advantage of this new feature?

**Comment by t1m0thyj** [comment, marked as answer]
WDD doesn't currently support different themes on different virtual desktops, but feel free to open an issue requesting this enhancement. The support for virtual desktops that is mentioned in the changelog is to sync the same theme across multiple virtual desktops. The default behavior would be that the wallpaper only updates on the virtual desktop where WDD is launched which is undesirable.
